Bots/AuthServer.js

```diff
@@ -1,59 +1,52 @@
-"use strict";
-
-var cookieParser = require('cookie-parser');
-var session = require('express-session');
-var bodyParser = require('body-parser');
-var express = require('express');
-var app = express();
-var router = express.Router();
-var passport = require('passport');
-var TwitterStrategy = require('passport-twitter').Strategy;
-
-var sessinCredential = require('common/makeCredential')('session');
-var twitterCredential = require('common/makeCredential')('twitter');
-var AuthBot = require('models/AuthBot');
-var log = require('common/log');
-
-function AuthServer(){
-
-  passport.serializeUser(function(user, done) {
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import session from 'express-session';
+import bodyParser from 'body-parser';
+import passport from 'passport';
+import { Strategy } from 'passport-twitter';
+import AuthBot from 'models/AuthBot';
+import log from 'common/log';
+
+const app = express();
+const router = express.Router();
+const sessinCredential = process.env.KINDLIZED_SESSION;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).